content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Gerenciando Secret usando Arquivo de Configuração
+content_type: task
+weight: 20
+description: Criando objetos Secret usando arquivos de configuração de recursos.
+---
+
+<!-- overview -->
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+<!-- steps -->
+
+## Crie o arquivo de configuração
+
+Você pode criar um Secret primeiramente em um arquivo, no formato JSON ou YAML, e depois
+criar o objeto. O recurso [Secret](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#secret-v1-core)
+contém dois *maps*: `data` e `stringData`.
```

Comment:
maybe maps should be in italic (_maps_) if we decide not to translate it. But I guess here it makes some sense to have "mapas" as this is a common development word translated to portuguese